modules/create_timepoint/php/ajaxtimepoint.class.inc

```diff
@@ -115,11 +119,12 @@ class AjaxTimepoint extends \NDB_Page
     {
         // Verify cand ID.
         $values['identifier'] = isset($values['identifier']) ?
-            htmlentities(
+            intval(htmlentities(
```

Comment:
You can just use `intval()` here instead of wrapping `htmlentities` because it'll only ever give you an integer.